tests/install-infra.sh

```diff
@@ -94,7 +94,15 @@ function test_obi {
 		for i in $(seq 1 2); do
 			[ $(kubectl get obi ${name} -n ${ARBITER_NS} -oyaml | grep 'timestamp' | awk '{print NR}' | tail -n1) -ge 2 ] && s=0 && break || s=$? && sleep 10
 		done
-		if [ $s -ne 0 ]; then
+
+		# check data valid
+		sum=$(kubectl get obi ${name} -n ${ARBITER_NS} -o jsonpath="{.status.metrics.*[*].records.*.value}" | awk 'BEGIN{sum=0} {for(i=1;i<=NF;i++)sum+=$i} END{print(sum)}')
+		if [ $sum -eq 0 ]; then
+			# The value is not a number and cant be summed does not means does not mean that it is wrong, perhaps the value is original raw data from prometheus
```

Comment:
The value is not a number and cant be summed does not means does not mean that it is wrong
=> The value might not be a number and  summed, but it maybe not wrong as the value maybe original raw data from prometheus.

Reply:
Done